### PR TITLE
Move FIXME to tech debt. Talk about not creating interfaces to soon.

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
=====

This is a small update on the styleguide to make it explicit that FIXME marker needs a comment and explain why.

I have also add some notes about not trying to create reusable code unless you have at least 3 places which are using it.

How to try and test the changes
===============================

reviewers: @brunogola @alibotean @lesingerouge @atykhonov 

check that changes are ok.

feel free to fix typos and commit them

Thanks!

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

**Comment by adiroiban** [comment]
forgot to add @atykhonov

File: content/defects-tech-debt.rst [hunk 1]
```diff
@@ -145,6 +140,54 @@ re-usable component itself so that it provide a clean re-usable API.
...
+
+    # FIXME:1234:
+    # Details about this tech-dept. Ex: Can only be fixed when full moon.
+
+    /* FIXME:1234:
+    Some other type of syntax.
+    */
+
+
+The comment should be descriptive enough so that so that when you are
```
Comment (alibotean) [inline content/defects-tech-debt.rst]: ~~so that~~

**Comment by alibotean** [comment]
changes-approved

**Comment by brunogola** [comment]
changes-approved

**Comment by adiroiban** [comment]
thanks for the review. going to merge it

**Pull request merged**